rclcpp/include/rclcpp/node_options.hpp

```diff
@@ -232,6 +232,24 @@ class NodeOptions
   NodeOptions &
   start_parameter_services(bool start_parameter_services);
 
+  /// Return the enable_log_service flag.
+  RCLCPP_PUBLIC
+  bool
+  enable_log_service() const;
+
+  /// Set the enable_log_service flag, return this for logger idiom.
+  /**
+   * If true, ROS services are created to allow external nodes to list, get,
```

Comment:
> If true, ROS services are created to allow external nodes to list, get,
> and request to set logger levels of this node.

If true, ROS services are created to allow external nodes to get and set logger levels of this node.